cavy.js

```diff
@@ -3,9 +3,34 @@ const program = require('commander');
 const init = require('./src/init');
 const runTests = require('./src/runTests');
 
-function getCommandParams(command, args) {
-  const commandIndex = args.indexOf(command);
-  return args.slice(commandIndex, args.length);
+function getCommandArgs(cmd) {
+  // Get array of Cavy options (short and long).
+  const options = cmd.options.map(option => [option.short, option.long]);
+  // Get array of all command line args.
+  const allArgs = process.argv;
+  const commandIndex = allArgs.indexOf(cmd.name());
+  const args = allArgs.slice(commandIndex, allArgs.length)
+
+  // Remove Cavy options from other args so RN cli doesn't try to call them.
+  options.forEach(option => {
+    for (var i = 0; i < args.length; i++) {
+      if (option.includes(args[i])) {
+        // Remove the option flag itself and its value.
+        args.splice(i, 2)
```

Comment:
How do you feel about double checking it ends in `.js` and there's a `.test.js` that matches?

Reply:
Shall we ignore typescript for now?

Reply:
I checked this in the `runTests` function itself rather than in cavy.js (this is responsible for higher-level program control).

Reply:
> Shall we ignore typescript for now? 😬

Yes, maybe an issue?